src/index.ts

```diff
@@ -108,9 +108,37 @@ export type RectangleAnnotation = {
   }
 }
 
+export type TextAnnotation = {
+  type: 'text'
+  id: string
+  x: number
+  y: number
+  width: number
+  height: number
+  content: string
+  resize?: boolean
+  boxStyle: {
```

Comment:
to make this more consistent with other interfaces, can we do
```ts
{
  ...,
  style: {
    color: string,
    stroke?: { ... } // does it make sense to make stroke optional?
    text?: { ... }
  }
}
```

Reply:
oh, should we also make the padding configurable? if that's too much of a pain, imho we should make it bigger by default, e.g. 8px